var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -96,6 +95,20 @@
     state: directory
     mode: 0777
 
+#
+# Local mounts should always run before the 'zfs-import' service since
+# that service will grab the 'spa_namespace_lock' and hold it for a long time.
+# This can result in a systemd timing out and a failure of the local mount
+# service. In addition, we need to run before the 'kdump-tools' service
+# since that service needs to write into '/var/crash'.
+#
+- mount:
+    path: /var/crash
+    src: rpool/crashdump
+    fstype: zfs
+    opts: x-systemd.before=zfs-import-cache.service,x-systemd.before=kdump-tools.service
```

Comment:
tangential question.. do we need these mount options for our other datasets managed by `/etc/fstab`?

Reply:
We need one of them and I'm filing a PR for it now.

Reply:
Nice, thank you.

Reply:
I just realized that I did not include `defaults` in the options list so I'll update this PR